app/assets/stylesheets/kaui/kaui.less

```diff
@@ -1,3 +1,5 @@
+@import "twitter/bootstrap/bootstrap";
```

Comment:
I had problems with the mixins using the other import. But I removed the duplication leaving only this one (and renaming `bootstrap_and_overrides` to `bootstrap_overrides`)